config/environments/production.rb

```diff
@@ -71,5 +71,12 @@
     config.logstasher.enabled = true
     config.logstasher.logger = Logger.new("#{Rails.root}/log/#{Rails.env}.json.log")
     config.logstasher.suppress_app_logs = true
+  else
```

Comment:
Unless-else seems to be considered as a something that reads badly. It would be better to invert the conditional and have it as if - else.